src/main/java/lv/acodemy/homework/PezDispenser.java

```diff
@@ -0,0 +1,113 @@
+package lv.acodemy.homework;
+
+import scala.collection.parallel.ParIterableLike;
+
+public class PezDispenser {
+
+    private String color;
+    private String pezName;
+    private String seriesName;
+    private int candyCount;
+    private final int MAX_CANDIES = 12;
+
+    public PezDispenser(String color, String pezName, String seriesName) {
+        this.color = color;
+        this.pezName = pezName;
+        this.seriesName = seriesName;
+        this.candyCount = MAX_CANDIES;
+    }
+    public String getColor() {
+        return color;
+    }
+    public String getPezName() {
+        return pezName;
+    }
+    public String getSeriesName() {
+        return seriesName;
+    }
+    public int getCandyCount() {
+        return candyCount;
+    }
+    public int getMax_Candies() {
+        return MAX_CANDIES;
+    }
+
+    // - отдавать по одной конфетке
+    public void eatCandy() {
+        if (candyCount == 0) {
+            System.out.println("Empty Pez Dispenser.");
+            System.out.println("Max is: 12. Current: " + candyCount);
```

Comment:
Not necessary to print amount of candies, because you already said that is it empty :)